apps/OboeTester/app/src/main/cpp/NativeAudioContext.cpp

```diff
@@ -299,6 +299,16 @@ oboe::Result ActivityContext::start() {
     return result;
 }
 
+oboe::Result ActivityContext::flush() {
+    oboe::Result result = oboe::Result::OK;
+    stopBlockingIOThread();
```

Comment:
This is a nice protection in a safe app.
But OboeTester should allow users to shoot themselves in the foot.
What happens if you flush a stream while another thread is writing to it?
Can we remove this stopBLockingIOThread()?
If it leads to a crash then we would like to understand why.

Reply:
I removed it. Flush returns AAUDIO_ERROR_INVALID_STATE when the stream is running for AAudio so this is fine. Somehow, OpenSL ES output streams just stop but don't crash. Oboe doesn't implement flush for OpenSL ES input streams.